src/screens/ProfileScreen.tsx

```diff
@@ -115,6 +114,11 @@ const ProfileScreen: FC<ProfileStackProfileProps> = () => {
     return <ActivityIndicator />;
   }
 
+  if (!currentMember || !('id' in currentMember)) {
+    // TODO !!!!!!! return an empty screen?
+    return null;
+  }
```

Comment:
What should I do here? I see error returns null, but it should display a full screen right? @alvrba

Reply:
In fact if you return null it will display an empty screen (keeping the app header and footer). Be careful because on line 110 you are checking the same condition `!currentMember` rendering the `<ActivityIndicator />`